great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -69,6 +74,36 @@ def _validate_builder_override_config(builder_config: dict):
         )
 
 
+class ReconciliationStrategy(Enum):
+    NESTED_UPDATE = "nested_update"
+    REPLACE = "replace"
+    UPDATE = "update"
+
+
+@dataclass
+class ReconciliationDirectives(SerializableDictDot):
+    variables: ReconciliationStrategy = ReconciliationStrategy.UPDATE
+    domain_builder: ReconciliationStrategy = ReconciliationStrategy.UPDATE
+    parameter_builder: ReconciliationStrategy = ReconciliationStrategy.UPDATE
+    expectation_configuration_builder: ReconciliationStrategy = (
+        ReconciliationStrategy.UPDATE
+    )
+
+    def to_dict(self) -> dict:
+        return asdict(self)
+
+    def to_json_dict(self) -> dict:
+        return convert_to_json_serializable(data=self.to_dict())
+
+
+DEFAULT_RECONCILATION_DIRECTIVES: ReconciliationDirectives = ReconciliationDirectives(
+    variables=ReconciliationStrategy.UPDATE,
+    domain_builder=ReconciliationStrategy.UPDATE,
+    parameter_builder=ReconciliationStrategy.UPDATE,
+    expectation_configuration_builder=ReconciliationStrategy.UPDATE,
+)
```

Comment:
Does this need to be a global value? What are your thoughts on making these class attributes? I'd rather not pollute the global scope if possible.

Reply:
@cdkini They will be used from outside RBP.  So, for that reason, I made them at the module level.  Thanks.